src/views/widgets/list-widget.php

```diff
@@ -56,9 +56,25 @@
 					 * @param $size
 					 */
 					$thumbnail_size = apply_filters( 'tribe_events_list_widget_thumbnail_size', 'post-thumbnail' );
+
+					/**
+					 * Filters whether the featured image link should be added to the Events List Widget
+					 *
+					 * @since TBD
+					 * @param bool Whether the featured image link should be added or not
+					 */
+					$featured_image_link = apply_filters( 'tribe_events_list_widget_featured_image_link', true );
 					?>
 					<div class="tribe-event-image">
-						<?php the_post_thumbnail( $thumbnail_size ); ?>
+						<?php if ( $featured_image_link ) : ?> 
+							<a href="<?php echo esc_url( tribe_get_event_link() ); ?>">
+						<?php endif; ?>
+							
+								<?php the_post_thumbnail( $thumbnail_size ); ?>
+
+						<?php if ( $featured_image_link ) : ?> 
+							</a>
+						<?php endif; ?>
```

Comment:
I'd recommend tidying this up a bit, perhaps something more like this:
```
<div class="tribe-event-image">

    <?php echo $featured_image_link ? '<a href="' . esc_url( tribe_get_event_link() ) . '">' : ''; ?>
				
    <?php the_post_thumbnail( $thumbnail_size ); ?>

    <?php echo $featured_image_link ? '</a>' : ''; ?>
</div>
```

Reply:
ternaries FTW!

Reply:
Or perhaps ditch the duplicated conditionals altogether:

```php
$featured_image_link = apply_filters( 'tribe_events_list_widget_featured_image_link', true );
$post_thumbnail      = get_the_post_thumbnail( null, $thumbnail_size );

if ( $featured_image_link ) {
  $post_thumbnail = '<a href="' . esc_url( tribe_get_event_link() ) . '">' . $post_thumbnail . '</a>';
}
?>
<div class="tribe-event-image">
  <?php
  // not escaped because it contains markup
  echo $post_thumbnail;
  ?>
</div>
```

Reply:
I'm putting my two cents in here but it's good the way it is (so I'm approving)
I like @borkweb's approach, but I'd use an if/else - and that may be down to personal preference:

```
$featured_image_link = apply_filters( 'tribe_events_list_widget_featured_image_link', true );


if ( $featured_image_link ) {
  $post_thumbnail = '<a href="' . esc_url( tribe_get_event_link() ) . '">' . $post_thumbnail . '</a>';
} else {
  $post_thumbnail = get_the_post_thumbnail( null, $thumbnail_size );
}
?>
<div class="tribe-event-image">
  <?php
  // not escaped because it contains markup
  echo $post_thumbnail;
  ?>
</div>
```